src/lib/containers/table/table-top/DownloadLoginModal.tsx

```diff
@@ -0,0 +1,78 @@
+import { Modal, ModalBody, ModalFooter, Button } from 'react-bootstrap'
+import Login from '../../Login'
+import ModalHeader from 'react-bootstrap/ModalHeader'
+import React, { useState } from 'react'
+
+export type DownloadLoginModalProps = {
+  showModal?: boolean
+  onHide: Function
+}
+
+export const DownloadLoginModal: React.FunctionComponent<DownloadLoginModalProps> = props => {
+  const [showLogin, setShowLogin] = useState(false)
+
+  const getModalContent = (isShowLogin: boolean): JSX.Element => {
+    if (!isShowLogin) {
```

Comment:
I find it a bit easier to read if the boolean is reversed in this specific case, so if (isShowLogin) return login else show content